### PR TITLE
Fix students admin panel, improved themes admin panel

### PR DESCRIPTION
**Pull request by Vlad1023**
1) Add confirmation before deleting theme
2) Add modal window to edit theme
3) Fix add button and field in AllThemes and AllStudents

### CODE REVIEW

File: CharlieBackend.AdminPanel/Views/Themes/AllThemes.cshtml [hunk 1]
```diff
@@ -81,4 +79,93 @@
...
+            editThemeName = _self.data('item-name');
+
+            $("#themeNameInput").val(editThemeName);
+        });
+        $("#themeNameInput").on('change keyup paste', function () {
+            $("#confirmEditTheme").attr("href", routeForEditTheme + editThemeId + '?&name=' + $("#themeNameInput").val());
+        });
+
+    </script>
+}
```
Comment (SnipeFrost) [inline CharlieBackend.AdminPanel/Views/Themes/AllThemes.cshtml:171]: add newline at end of file

**Review by SnipeFrost — APPROVED** [review]

File: CharlieBackend.AdminPanel/Controllers/ThemesController.cs [hunk 2]
```diff
@@ -56,14 +47,15 @@ public async Task<IActionResult> AddTheme(CreateThemeDto themeDto)
             return RedirectToAction("AllThemes", "Themes");
         }
 
-        [HttpPost("{id}")]  
-        public async Task<IActionResult> UpdateTheme(long id, UpdateThemeDto data)
+        [HttpGet("{id}")]
```
Comment (SerjKocherga) [inline CharlieBackend.AdminPanel/Controllers/ThemesController.cs:50]: Why are you using GET method on update logic?
Probably should be better to use HttpPut?

**Review by SerjKocherga — CHANGES_REQUESTED** [review]

File: CharlieBackend.AdminPanel/Controllers/ThemesController.cs (hunk 2 above)
Comment (Vlad1023) [reply CharlieBackend.AdminPanel/Controllers/ThemesController.cs:50]: unfortunately it`s not possible, because we can`t use HttpPut  in this case

Comment (Vlad1023) [reply CharlieBackend.AdminPanel/Controllers/ThemesController.cs:50]: <a> tag supports only get method

**Review by SerjKocherga — APPROVED** [review]

**Review by BohdanBBG — APPROVED** [review]

**Review by sober607 — APPROVED** [review]

File: CharlieBackend.AdminPanel/Views/Themes/AllThemes.cshtml (hunk 1 above)
Comment (Vlad1023) [reply CharlieBackend.AdminPanel/Views/Themes/AllThemes.cshtml:171]: done

**Review by SnipeFrost — APPROVED** [review]

**Review by x1mx1x — APPROVED** [review]

**Pull request merged**